src/tests/timesync/TimeSync_SyncTime_001.test.js

```diff
@@ -0,0 +1,29 @@
+import { getLatestSyncTime } from '../../commonMethods/commonFunctions'
+import baseTest from './TimeSync_Synchronize_001.test'
+
+export default {
+  ...baseTest,
+  ...{
+    title: 'TimeSync - SyncTime 001',
+    description: 'Check the latest Synchronized time',
+    steps: [
+      baseTest,
+      {
+        description: 'Get latest Synchronized time',
+        sleep: 5,
+        test() {
+          return getLatestSyncTime.call(this)
+        },
+        validate(res) {
+          //TODO - Need to update the validation with most recent Synced time
```

Comment:
recommend to use momentjs or similar, just check if the date (day month year) and hours / minutes are the same to avoid seconds / miliseconds differences and slippage.